modules/mono/editor/GodotTools/GodotTools/Build/BuildSystem.cs

```diff
@@ -176,8 +176,9 @@ private static void BuildPublishArguments(BuildInfo buildInfo, Collection<string
                 arguments.Add("--no-restore");
 
             // Incremental or rebuild
-            if (buildInfo.Rebuild)
-                arguments.Add("--no-incremental");
+            // TODO: Not supported in `dotnet publish` (https://github.com/dotnet/sdk/issues/11099)
+            // if (buildInfo.Rebuild)
+            //     arguments.Add("--no-incremental");
```

Comment:
Weird, we shouldn't be using Rebuild when publishing.

Reply:
Oh it wasn't, I just noticed that the option doesn't exist in `dotnet publish`

Reply:
My bad, I didn't notice the comment about the issue and that this was the `BuildPublishArguments` method.